extension/src/plots/webview/messages.ts

```diff
@@ -285,7 +297,8 @@ export class WebviewMessages {
       nbItemsPerRow: this.plots.getNbItemsPerRowOrWidth(
         PlotsSection.TEMPLATE_PLOTS
       ),
-      plots
+      plots,
+      smoothPlotValues: this.plots.getSmoothPlotValues()
```

Comment:
[Q] What was the reason for keeping this separate from the plots data?

Reply:
Didn't think of it :woman_facepalming:. That would probably make updating the plots' vega state easier too.

Reply:
Ended up keeping it separate for now so we could watch for changes to values by just watching the values instead of `snapshots` which would cause unnecessary checks.